crypto/keys/mintkey/README.md

```diff
@@ -0,0 +1,36 @@
+Security parameter choice
+-------------------------
+
+The present Bcrypt security parameter used is 12, which should take about a quarter of a second on midrange consumer hardware (see [Benchmarking](#benchmarking) section below).
```

Comment:
Should we add a note/sentence on on how many bits of security this provides us?

Reply:
Bcrypt generates a 448-bit key, but in practice the bits of security are determined by the length of a user's password, since that's what an adversary would try to brute-force (or use a dictionary attack to guess) - hence the concern about time taken to hash a password.

Reply:
I've added a line about this, good call.